kmq.js

```diff
@@ -56,14 +56,17 @@ client.on("voiceStateUpdate", (oldState, newState) => {
             let voiceConnection = client.voice.connections.get(guildID);
             if (voiceConnection) {
                 voiceConnection.disconnect();
-                gameSession.endRound();
+                if (gameSession) {
```

Comment:
Why not check for `!gameSession` on line 54 and do early return instead

Reply:
Then `.disconnect()` won't be called, right?

Reply:
Whoops, I don't think it should be possible.